docs/introduction-to-smart-contracts.rst

```diff
@@ -256,14 +257,17 @@ Ethereum this is roughly every 17 seconds.
 
 As part of the "order selection mechanism" (which is called "mining") it may happen that
 blocks are reverted from time to time, but only at the "tip" of the chain. The more
-blocks that are added on top, the less likely it is. So it might be that your transactions
+blocks that are added on top, the less likely this block will be reverted. So it might be that your transactions
 are reverted and even removed from the blockchain, but the longer you wait, the less
 likely it will be.
 
 .. note::
-    Transactions are not guaranteed to happen on the next block or any future specific block, since it is up to the miners to include transactions and not the submitter of the transaction. This applies to function calls and contract creation transactions alike.
+    Transactions are not guaranteed to happen on the next block or any future specific block,
+    since it is up to the miners to include transactions and not the submitter of the transaction.
```

Comment:
``since it is not up to the submitter of a transaction, but up to the miners, in which block the transaction is included.``?